### PR TITLE
Use absolute imports in Runner

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Within eq-questionnaire-runner, a number of files and test cases were using relative imports.

Installed the `pylint-absolute-imports` package to ensure that absolute imports are enforced during the pylint check.

### How to review

Check that absolute imports are being enforced.

Check nothing has been missed.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Update Runner tests to use absolute imports (1 SP)](https://trello.com/c/xkdluscU/5780-update-runner-tests-to-use-absolute-imports-1-sp)

File: Pipfile.lock [hunk 1]
```diff
@@ -628,69 +639,67 @@
...
+                "sha256:ef7c6e49a9a020d56349c6a769352709bfbe35d3ee7f98bd5efcac6cedbdc162",
+                "sha256:ef9789aea735004eba559c7919a73a3b475d0c28e2c1e9de464c6bc761bf69f4",
+                "sha256:f059457db4e2ae4a4fdae455453c5e5765aa08efcb804e2a106c69c31bd438ba",
+                "sha256:f0c5508582339090b99e2863a157fc2708ab9c8b5cd21619bdcb04edcdc6c28d",
+                "sha256:f1c9ad8d6500f7b142a94054281d9628bc8652a14b0923d02e0dfd87392fbc74",
+                "sha256:f33e7ff85775cb0ec6abb0950ffc631960bae5a203da38166fc3dfde826e0d0a",
+                "sha256:f8661d14d3e07f2ceeb850e4cbcc7114bdf90a8dc82d63d37b08a50bb6955a77"
+            ],
+            "markers": "platform_python_implementation == 'CPython' and python_version >= '3.11'",
+            "version": "==3.0.0rc3"
```
Comment (MebinAbraham) [inline Pipfile.lock:702]: This is a pre-release candidate and not really recommended for production use until released. We should pin the lib that forces this upgrade.

**Review by MebinAbraham — APPROVED** [review]
Import changes look good, but the package update needs re-visiting.

File: Pipfile.lock (hunk 1 above)
Comment (berroar) [reply Pipfile.lock]: Pinned the gevent library that was responsible for this upgrade 👍

File: .pylintrc [hunk 2]
```diff
@@ -77,7 +77,7 @@ limit-inference-results=100
 
 # List of plugins (as comma separated values of python module names) to load,
 # usually to register additional checkers.
-load-plugins=pylint.extensions.mccabe,pylint_quotes,pylint.extensions.no_self_use
+load-plugins=pylint.extensions.mccabe,pylint_quotes,pylint.extensions.no_self_use,pylint_absolute_imports
```
Comment (katie-gardner) [inline .pylintrc:80]: Good shout 👍

**Review by katie-gardner — APPROVED** [review]

**Review by petechd — APPROVED** [review]

**Pull request merged**